packages/contracts/localhost_tests_OP/test-general-tokens.spec.ts

```diff
@@ -335,6 +309,117 @@ makeSuite(
+            var USDCABI = fs.readFileSync("./localhost_tests/abis/DAI_ABI.json").toString()
+            var yvUSDC = new ethers.Contract(USDCadd,USDCABI)
+            const yvUSDCDat = await lendingPool.getReserveData("0xaD17A225074191d5c8a37B50FdA1AE278a2EE6A2", 0);
+            increaseTime(50000)
+            console.log("How much yvUSDC is held in aToken: ", await yvUSDC.connect(signer).balanceOf(yvUSDCDat.aTokenAddress))
+            console.log("How much yvUSDC is held in incentives controller: ", await yvUSDC.connect(signer).balanceOf(incentivesController.address))
+            const stakingContract = new ethers.Contract("0xB2c04C55979B6CA7EB10e666933DE5ED84E6876b", Stakingabi);
+            const amtStaked = await stakingContract.connect(signer).balanceOf(incentivesController.address)
+            const earned = await stakingContract.connect(signer).earned(incentivesController.address)
+            console.log("earned: ",earned)
+            expect(amtStaked).equal(ethers.utils.parseUnits("9.0", 6))
+            expect(earned).gt(0)
+
+            const OP = new ethers.Contract("0x7D2382b1f8Af621229d33464340541Db362B4907", WETHabi)
+            const balanceBefore = await OP.connect(signer).balanceOf(incentivesController.address);
+            const receipt = await waitForTx(
+              await incentivesController.harvestReward("0xB2c04C55979B6CA7EB10e666933DE5ED84E6876b", 3, ["0x7D2382b1f8Af621229d33464340541Db362B4907"])
+            );
+
+            const balanceAfter = await OP.connect(signer).balanceOf(incentivesController.address);
+            const reward = Number(balanceAfter) - Number(balanceBefore);
+            console.log("true rewards earned: ", reward);
+            console.log("earned: ", await stakingContract.connect(signer).earned(incentivesController.address));
+            const emitted = receipt.events || [];
+
+            eventChecker(emitted[2], 'HarvestedReward', [
+              "0xB2c04C55979B6CA7EB10e666933DE5ED84E6876b",
+                ["0x7D2382b1f8Af621229d33464340541Db362B4907"],
+                [reward]
+            ]);
+          });
+
+
+          it("wait and harvest velodrome rewards", async () => {
+            var signer = await contractGetters.getFirstSigner();
+            const incentivesController = await contractGetters.getIncentivesControllerProxy();
+            const veloAdd = "0x3c8B650257cFb5f272f799F5e2b4e65093a11a05"
+            
+            const stakingContract = new ethers.Contract("0x131Ae347E654248671Afc885F0767cB605C065d7", StakingVeloabi); //staking for velo_wstETHWETH
+            const amtStaked = await stakingContract.connect(signer).balanceOf(incentivesController.address)
+            const earned = await stakingContract.connect(signer).earned(veloAdd, incentivesController.address)
+            console.log("earned: ",earned)
+            expect(amtStaked).equal(ethers.utils.parseUnits("9.0", 18))
+            expect(earned).gt(0)
```

Comment:
I dont see waiting in the test, how is this generating rewards?

Reply:
Waits in yearn one